Tests/test_SeqIO_QualityIO.py

```diff
@@ -376,7 +422,9 @@ def test_qual_negative(self):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", BiopythonParserWarning)
             self.assertEqual(4, SeqIO.convert(h, "qual", h2, "fastq"))
-        self.assertEqual(h2.getvalue(), """@1117_10_107_F3
+        self.assertEqual(
+            h2.getvalue(),
+            """@1117_10_107_F3
```

Comment:
Perhaps this is nicer?

```python
        self.assertEqual(
            h2.getvalue(),
            """\
@1117_10_107_F3
...
```

See discussion on #2369

Reply:
yes, looks better, they harder to spot, amended, Thank you.